Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/CarbonBlackLiveResponseCloud.py

```diff
@@ -0,0 +1,319 @@
+
+    headers = ['path', 'pid', 'command_line', 'username']
+    processes_readable = [dict(
+        path=process['path'],
+        pid=process['pid'],
+        command_line=process['command_line'],
+        user_name=process['username']) for process in processes]
+    context_entry = dict(sensor_id=sensor_id, processes=processes)
+
+    readable_output = tableToMarkdown('Carbon Black Defense Live Response Processes',
+                                      headers=headers,
+                                      t=processes_readable,
+                                      headerTransform=string_to_table_header,
+                                      removeNull=True)
+    return CommandResults(
+        'CarbonBlackDefenseLR.Processes',
+        outputs_key_field='sensor_id',
+        outputs=context_entry,
+        readable_output=readable_output,
+        raw_response=processes
+    )
+
+
+# Using Py API
+def kill_process_command(credentials: Dict, sensor_id: int, pid: Any):
+    api = CBCloudAPI(**credentials)
+
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    res = session.kill_process(pid)
+    if res:
+        return f'The process: {pid} was killed'
+
+    return f'Can not kill the process: {pid}'
+
+
+# Using Py API
+def create_process_command(
+        credentials: Dict, sensor_id: int,
+        command_string: str,
+        wait_timeout: int = 30,
+        wait_for_output: bool = True,
+        wait_for_completion: bool = True,
+        **additional_params):
+    # additional_param may include: remote_output_file_name: str, working_directory: str
+    api = CBCloudAPI(**credentials)
+
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    res = session.create_process(
+        command_string=command_string,
+        wait_timeout=int(wait_timeout),
+        wait_for_output=str(wait_for_output) == 'True',
+        wait_for_completion=str(wait_for_completion) == 'True',
+        **additional_params)
+    if res:
+        return res
+    return f'Command: {command_string} was executed'
+
+
+# Using Py API
+def memdump_command(credentials: Dict, sensor_id: int, target_path: str):
+    api = CBCloudAPI(**credentials)
+
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    session.start_memdump(remote_filename=target_path).wait()
+    return f'Memory was dumped to {target_path}'
+
+
+def command_test_module(credentials: Dict) -> str:
+    message: str = ''
+    try:
+        api = CBCloudAPI(**credentials)
+        api.api_json_request(method='GET', uri='/integrationServices/v3/cblr/session/')
+        message = 'ok'
+
+    except errors.UnauthorizedError:
+        message = 'Authorization Error: Check your API Credentials'
+    except Exception as e:
+        exception_str = str(e)
+        if 'connection error' in exception_str:
+            message = 'Connection Error: Check your Server Url'
+        else:
+            raise e
```

Comment:
You should use `return_error` with a more descriptive error, that will include the original error message.
Something like: `f'An error occurred.\n{exception_str}'`